src/pages/UnlockedWallet/AddressesPage/index.tsx

```diff
@@ -103,67 +103,69 @@ const AddressesPage = () => {
           </Tabs>
         </MainPanel>
       </Scrollbar>
-      <AdvancedOperationsPanel>
-        <AnimatePresence>
-          {isAdvancedSectionOpen && (
-            <BottomModalStyled
-              onClose={() => setIsAdvancedSectionOpen(false)}
-              label={t`Advanced operations`}
-              // TODO: Is there a better way than passing a hardcoded height?
-              contentHeight={344}
-            >
-              <AdvancedOperations>
-                <OperationBox
-                  title={t`Consolidate UTXOs`}
-                  Icon={<Codesandbox color="#64f6c2" strokeWidth={1} size={46} />}
-                  description={t`Consolidate (merge) your UTXOs into one.`}
-                  buttonText={t`Start`}
-                  onButtonClick={() => setIsConsolidationModalOpen(true)}
-                  infoLink={links.utxoConsolidation}
+      {currentTab.value === 'addresses' && (
```

Comment:
Mhh this makes me think about the UI hierarchy again. We don't have any advanced actions linked to contacts, but in theory, changing a tab shouldn't have any impact on anything out of the tab's "folder". Don't know if I'm clear.

Maybe I need to redesign this "folder" a bit. To be continued...

Reply:
Something like this (quick and dirty) :

<img width="948" alt="image" src="https://user-images.githubusercontent.com/6715943/213138741-7be290f5-29ec-4b88-80a6-7d8d5476fde9.png">

WDYT?

Reply:
I know exactly what you mean. I am not 100% sold on the vertical scrolling, but we could try it. Should it be part of this PR?

Reply:
Let's try the vertical scrolling and see how it feels. If we end up going for horizontal scrolling, I may switch from cards to "rows" to ease visual parsing.

We can tweak the layout in another PR!

Reply:
So, here's a quick and dirty solution. What I don't like about it is when the height of the page is not big enough and it requires its own scrolling. If we remove vertical scrolling of the page and force a fixed `800px` height, then our wallet is not gonna work well in smaller screens.

I find it very annoying to have both veritical and horizontal scroll in the page, because it forces me to move the mouse to an area outside of the "horizontally-scrollable" area so that I can scroll vertically.

I think horizontal scrolling works great on mobile devices. On desktop, I am not a big fan. I think the mental model is that scrolling is always vertical. 

But we don't have to do what the status quo is, we can break things :) 

Lemme know what you think.


https://user-images.githubusercontent.com/1579899/213237238-ed16f5b1-e097-4f13-8e5b-360a5ebd2fcb.mp4

Reply:
Thanks for this. Yeah I'm not 100% convinced yet.
What we could do is to have a switch from card view to list view in the header (like I imagined for the mobile wallet as well). Something like: 
I'd like to design an alternative "vertical list" version that is not a cold and dense table... Let's see when I can do that!

Reply:
Alright, I'll move on to other screens then and come back to this at a later point! Thanks for the discussion!